lib/repositories/movie_repository.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+
+import '../entities/movie.dart';
+
+class MovieRepository {
+  Future<List<Movie>> getMovies() async {
+    final String response =
+        await rootBundle.loadString('assets/mock_data/movies.json');
+    List<dynamic> data = List.from(json.decode(response));
+    return data
+        .map(
+          (movie) => Movie.fromJson(
+            movie,
+          ),
+        )
```

Comment:
In this case, I believe that these trailing commas are not needed

Reply:
done